src/deluge/dsp/reverb/base.hpp

```diff
@@ -14,12 +14,25 @@ struct Base {
 		amplitude_right_ = amplitude_right;
 		amplitude_left_ = amplitude_left;
 	}
-
-	static constexpr float calcFilterCutoff(float f) {
+	enum class FilterType {
+		LowPass,
+		HighPass
+	};
+	static constexpr float calcFilterCutoff(float f, FilterType filtertype) {
```

Comment:
```suggestion
	template <FilterType filtertype>
	static constexpr float calcFilterCutoff(float f) {
```
Instead of passing FilterType as a parameter, this should be a template parameter so that the branches below can be constexpr and not eval'd at runtime

Reply:
Oh nice, I've been trying to find a clean way to do this